bin/impl/fetch.sh

```diff
@@ -59,10 +59,14 @@ function fetch_accumulo() {
   fi
 
   if [[ -n "$ACCUMULO_REPO" ]]; then
+    declare -a maven_args=(-DskipTests -DskipFormat)
+    if [[ "${HADOOP_VERSION}" = 3.* ]]; then
+      maven_args=("${maven_args[@]}" '-Dhadoop.profile=3')
```

Comment:
Accumulo 2 is Hadoop 3 only.  So this would be unnecessary, but I don't think it would hurt anything.

Reply:
I agree it won't hurt anything to define the variable and not have it used in the Accumulo 2 builds. I'm fine for being precise though, just let me know if you want to see an update.

Reply:
I think its fine.  Looking at these changes made me think it would be nice if Uno failed fast  if trying to run Accumulo 1.x with Hadoop 3 when not building Accumulo.

Reply:
Or running accumulo 2 when the Hadoop version isn't 3+

Reply:
Created #198